src/cell.cpp

```diff
@@ -578,8 +584,12 @@ CSGCell::to_hdf5(hid_t cell_group) const
       write_dataset(group, "translation", translation_);
     }
     if (!rotation_.empty()) {
-      std::array<double, 3> rot {rotation_[0], rotation_[1], rotation_[2]};
-      write_dataset(group, "rotation", rot);
+      if (rotation_.size() == 12) {
+        std::array<double, 3> rot {rotation_[9], rotation_[10], rotation_[11]};
```

Comment:
im sure this is obvious, but why not preserve the raw rotation matrix? in from_hdf5 and to_hdf5?

Reply:
Not exactly obvious -- doing it this way preserves the user's original intent by storing the angles in the file when they provided angles. Not sure it matters much, but it does keep the current behavior the same.